test/new-e2e/tests/containers/docker_test.go

```diff
@@ -0,0 +1,112 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package containers
+
+import (
+	"context"
+	"fmt"
+	fakeintake "github.com/DataDog/datadog-agent/test/fakeintake/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/infra"
+	dockervm "github.com/DataDog/test-infra-definitions/scenarios/aws/dockerVM"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/stretchr/testify/suite"
+	"os"
+	"testing"
+)
+
+type DockerSuite struct {
+	baseSuite
+	stackName string
+}
+
+func TestDockerSuite(t *testing.T) {
+	stackName := "docker-stack"
+	suite.Run(t, &DockerSuite{stackName: stackName})
+}
```

Comment:
I don’t think we need the `stackName` field in the `DockerSuite` struct as it’s always set to the same hard-coded value and it’s never modified.

```suggestion
type DockerSuite struct {
	baseSuite
}

func TestDockerSuite(t *testing.T) {
	stackName := "docker-stack"
	suite.Run(t, &DockerSuite{})
}
```

Reply:
in fact I added it as a property on the suite because it is used 3 times through out the code. 

So thought would be better to have a source value on the suite object instead of hardcoding the value several times.

Why do you think it is better to hard-code the stackname several times compared to having it as a property on the suite?

Reply:
I can find this stack name used only once as a parameter of `infra.GetStackManager().GetStack(…` line 39.
Where are the other 2 usage locations ?

Anyway, if we really have several places where the same value must be used, it would be better to define a `const`.

Reply:
Yes indeed, the other usages had been removed after committing your other suggestion. Sorry for the confusion.